api/src/data/models/index.ts

```diff
@@ -1,2 +1,27 @@
+export * from "./action-attachment-model";
```

Comment:
Avoid doing `export * from 'xxx'` as it will lead to hard to find name collisions.
Instead do direct exports like
```
export { ActionAttachment } from "./action-attachment"
```

If you do need to export a bundle; name the bundle.

e.g.
```
export * as ActionAttachments from "./action-attachments"
```
This would create the "ActionAttachments" namespace that contains all the "action attachment sub-models".